packages/api/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es6",
+    "target": "es2017",
```

Comment:
This is a big change, and I'd rather not research the implications unless absolutely necessary...